app/views/ShareView/index.js

```diff
@@ -81,6 +85,11 @@ export default class ShareView extends React.Component {
 		navigation.setParams({ sendMessage: this._sendMessage, canSend: !(isReadOnly(room, { username }) || isBlocked(room)) });
 	}
 
+	componentDidUpdate() {
+		const { setSearch } = this.props;
+		setSearch('');
```

Comment:
Bad practice.
You executing `setSearch('')` for every render.
Is this what you want to do here?